packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.routing.js

```diff
@@ -18,6 +18,41 @@ export default /* @ngInject */ ($stateProvider) => {
         $transition$.params().productId,
       status: /* @ngInject */ ($transition$) => $transition$.params().status,
       path: /* @ngInject */ ($transition$) => $transition$.params().path,
+      isConfiguration: () => false,
+      sshKey: /* @ngInject */ (
+        HostingMultisiteGitAssociationService,
+        serviceName,
+      ) => HostingMultisiteGitAssociationService.getSshKey(serviceName),
+      webHookUrl: /* @ngInject */ (
+        HostingMultisiteGitAssociationService,
+        serviceName,
+        path,
+      ) =>
+        HostingMultisiteGitAssociationService.getVcsWebhookUrls(
+          serviceName,
+          path,
+          GITHUB_VCS,
+        ).then(({ push }) => push),
+    },
```

Comment:
for avoid the duplicate of code you can create an object "common resolve" between association and configuration there are some example in code base

Reply:
ah yes good catch!